yt_dlp/extractor/tiktok.py

```diff
@@ -68,6 +69,9 @@ def _call_api(self, ep, query, video_id, fatal=True,
             'cp': 'cbfhckdckkde1',
         }
         self._set_cookie(self._API_HOSTNAME, 'odin_tt', ''.join(random.choice('0123456789abcdef') for _ in range(160)))
+        webpage_cookies = self._get_cookies(self._WEBPAGE_HOST)
+        if webpage_cookies and webpage_cookies.get('sid_tt'):
+            self._set_cookie(self._API_HOSTNAME, 'sid_tt', webpage_cookies.get('sid_tt').value)
```

Comment:
```suggestion
        if webpage_cookies.get('sid_tt'):
            self._set_cookie(self._API_HOSTNAME, 'sid_tt', webpage_cookies['sid_tt'].value)
```
_get_cookies alreays returns a dict